README.md

```diff
@@ -1,38 +1,22 @@
-# IBM Cloud SDK for Power Cloud
-
+# IBM Power-Go-Client
 The power-go-client project provides the Go SDK for IBM® Power Systems™ Virtual Server.
 
-## Prerequisites
-
-- An [IBM Cloud Account](https://cloud.ibm.com/registration)
-- An IBM Cloud [IAM API key](https://cloud.ibm.com/docs/account?topic=account-userapikey) or [Token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey) allow the SDK to access your account.
-- Go version 1.17 or above.
+## Requirements
+ - An [IBM Cloud Account](https://cloud.ibm.com/registration)
+ - An IBM Cloud [IAM API key](https://cloud.ibm.com/docs/account?topic=account-userapikey) or [Token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey) allow the SDK to access your account
+ - [Go](https://golang.org/doc/install) v1.17 or above.
+ - [Go Swagger](https://goswagger.io/install.html) v0.19.0
 
 ## Install
-
-Install the SDK using the following methods.
-
-### `go get` command
-
-Use this command to download and install the SDK to allow your Go application to use it:
-```
-go get -u github.com/IBM-Cloud/power-go-client
-```
-
-### Go modules
-
-If your application is using Go modules, you can add a suitable import to your Go application, like this:
-```go
-import (
-  "github.com/IBM-Cloud/power-go-client"
-)
-```
-then run `go mod tidy` to download and install the new dependency and update your Go application's
-`go.mod` file.
-
-### `dep` dependency manager
-
-If your application is using the `dep` dependency management tool, you can add a dependency
+ - Install the SDK
+    ```
+    go get github.com/IBM-Cloud/power-go-client
+    ```
+ - Update the SDK
+    ```
+    go get -u github.com/IBM-Cloud/power-go-client
+    ```
+ - If your application is using the `dep` dependency management tool, you can add a dependency
```

Comment:
Any reason of removing go modules method? Any reason to change the existing content?

Reply:
I re-added the original "install" section